api_EPA.py

```diff
@@ -23,12 +25,27 @@
 filename = "test.csv"
 file_path = os.path.join("./", filename)
 
+#case where the script is run twice back to back and the second time there aren't any results to show
+#if array length is 0 exit // no new data
+# end date at most 2 days later that the start time
 
 def main():
-    # TODO: get last_updated timestamp, fetch all data since
-    print("Fetching EPA data")
 
-    epa_data = pull_from_epa_api(start_time="2020-07-06T00", end_time = "2020-07-07T00")
+    #raw_end_time = strftime('%Y-%m-%dT%H:%M', gmtime())
+
+    start_time = get_start_time()
+
+    temp = get_start_time()
+
+    starting_day = int(temp[9])
+
+    end_day = starting_day + 2
+
+    temp.replace(temp[9], str(end_day))
+
+    epa_data = pull_from_epa_api(start_time=start_time, end_time=temp)
+
+    return
```

Comment:
Let's remove this return statement and validate that the whole script runs (that is, uploads the data and releases a new version)